exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -223,19 +223,65 @@ public void connect(Properties props) throws RpcException {
     connect(null, props);
   }
 
+  /**
+   * Function to populate the endpointList with information of all the drillbits
+   * provided in the connection string by client
+   * @param endpointList - ArrayList of DrillbitEndpoints
+   * @param drillbits - One or more drillbit ip[:port] provided in connection string
```

Comment:
Just to clarify the I am not changing the variable name "drillbit" used in the connection string. So there won't be any doc impact. The variable "drillbits" is used in the internal function which is parsing the string. 

As discussed the documentation is already there [here](http://drill.apache.org/docs/using-the-jdbc-driver/#using-the-jdbc-url-format-for-a-direct-drillbit-connection)